app/Http/Controllers/Auth/ApplicationController.php

```diff
@@ -153,14 +159,38 @@ public function editApplication(Request $request): RedirectResponse
     {
         $this->authorize('viewAnyApplication', User::class);
         $application = ApplicationForm::findOrFail($request->input('application'));
+        $newStatus=$request->input('status_'.$application->user->id);
         if ($request->has('note')) {
             $application->update(['note' => $request->input('note')]);
-        } elseif ($request->has('banish')) {
-            $application->update(['status' => ApplicationForm::STATUS_BANISHED]);
+        } elseif ($newStatus) {
+            $application->update(['status' => $newStatus]);
+            if ($newStatus==ApplicationForm::STATUS_CALLED_IN || $newStatus==ApplicationForm::STATUS_ACCEPTED) {
+                $application->user->internetAccess->setWifiUsername($application->user->educationalInformation->neptun??'wifiuser_'.$application->user->id);
```

Comment:
Just call setWifiUsername(). That should work. If not, thats an other issue :D

Reply:
I got an error saying $user was null, which I couldn't figure out, so I did this instead.